hack/test-go.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# Copyright 2014 The Kubernetes Authors.
+# Copyright 2016 The Kubernetes Authors.
```

Comment:
The file was moved with year intact and a new file with the same name was added.  git can't track that